be/src/star_cache/cache_item.h

```diff
@@ -0,0 +1,117 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+#pragma once
+
+#include <atomic>
+#include <shared_mutex>
+#include "star_cache/block_item.h"
+
+namespace starrocks {
+
+enum CacheState {
+    IN_MEM,
+    IN_DISK,
+    RELEASED
+};
+
+struct CacheItem {
+    // An unique id generated internally
+    // CacheId cache_id;
+    // A key string passed by api
+    std::string cache_key;
+    // The block lists belong to this cache item
+    BlockItem* blocks;
+    // The block count of this cache item
+    uint32_t block_count;
+    // The cache object size
+    size_t size;
```

Comment:
Is size = block_count * block_size ?

Reply:
> Is size = block_count * block_size ?

Not exactly. `size` means the accurate bytes of a cache item. If the last block is not full, `size` may smaller than `block_count * block_size`.